src/interpreter.jl

```diff
@@ -105,12 +105,22 @@ function InferredSource(state::InferenceState)
         exct)
 end
 
+@static if VERSION ≥ v"1.13.0-DEV.126" || v"1.12.0-alpha1" ≤ VERSION < v"1.13"
```

Comment:
The fix has been marked as a 1.12 backport so this check allows us to not break nightly versions for `v"1.13.0-DEV.0" < VERSION < v"1.13.0-DEV.126"`. On the other hand it's a bit complex, I'm not sure how we usually deal with backports in this situation.

Happy to change that (and the other check below) to `VERSION ≥ v"1.12.0-alpha1"` if the complex version management is not deemed worth it.

Reply:
We don't have a solid rule for this sort of things, but I think I'd prefer to having a simpler versioning here, so `VERSION ≥ v"1.12.0-alpha1"` would be best. We generally don't account those who use outdated nightly version.